### ISSUE TITLE
formatting issues

### ISSUE THREAD

**Issue by KABBOUCHI** [sajansharmanz/prettier-plugin-edgejs#4]
git diff after running prettier, `{` is missing, and  `@flashMessage` is inlined

```diff edge
<div>
-    {something}
+  something}
</div>
 ```
 
 ```diff edge
<div
-    @flashMessage('success')
-      data-message="{{ $message }}"
-    @end
+    @flashMessage success data-message="{{ $message }}" @end
>
</div>
 ```
 

  
example https://github.com/KABBOUCHI/adonisjs-livewire-pingcrm/blob/master/resources/views/livewire/flash-messages/show.edge

**Comment by sajansharmanz** [comment]
Hey,

Thanks for the bug.

1. flashMessage isn't yet included as a valid prop for html tags in the current setup. I will look to add this in now.

2. the {something} part is a bug, this should be treated as html text by the parser but it doesn't seem to be doing that. Will look into fixing this.

Will keep you updated on when it should be good to go.

**Comment by sajansharmanz** [comment]
The issues should now be fixed in prettier-edgejs@0.2.25
